kratos/python_scripts/assign_vector_by_direction_process.py

```diff
@@ -33,7 +34,6 @@ def __init__(self, Model, settings ):
         default_settings = KratosMultiphysics.Parameters("""
         {
             "help"                 : "This process sets a variable a certain scalar value in a given direction, for all the nodes belonging to a submodelpart. Uses assign_scalar_variable_to_conditions_process for each component",
-            "mesh_id"              : 0,
```

Comment:
Let me be the first to make one of these comments :smiling_imp: 

Please update documentation to reflect this change ;)

Reply:
Done!